### ISSUE TITLE
Enhancement: Allow Direct2D in WPF Platform

### ISSUE THREAD

**Issue by jjxtra** [picoe/Eto#425]
It would be nice to allow the WPF Platform to use Direct2D instead of WPF for paint events. As I don't know how involved this is, I'm happy to help out if this seems like a good idea.

http://www.codeproject.com/Articles/113991/Using-Direct2D-with-WPF

**Comment by cwensley** [comment]
Yes this is a great idea and I'd love to get support for it.  It should be doable, though integrating Direct2D with WPF might have a few challenges.  At the very least we could do it similarly to the (incomplete) WinRT platform, but ideally something more like the link you posted would be ideal.

**Comment by cwensley** [comment]
As mentioned in #425, this will be needed before #627 is implemented.
